rules-tests/TypeDeclaration/Rector/ClassMethod/StrictArrayParamDimFetchRector/Fixture/skip_possible_array_access_object.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictArrayParamDimFetchRector\Fixture;
+
+final class SkipPossibleArrayAccessObject
+{
+    public function resolve($item)
+    {
+        $item->doSomething();
+
+        return $item['name'];
```

Comment:
this might be a object which implements ArrayAccess and therefore has a array-dim-fetch